src/handlers/health_check.rs

```diff
@@ -0,0 +1,31 @@
+use actix_web::{web, HttpResponse};
+use serde::Serialize;
+use sqlx::postgres::PgRow;
+use sqlx::Row;
+
+use crate::state::State;
+
+pub async fn health_check(state: web::Data<State>) -> HttpResponse {
+    let value = sqlx::query("SELECT 1")
+        .try_map(|row: PgRow| row.try_get::<i32, _>(0))
+        .fetch_one(&state.db)
+        .await
+        .unwrap_or(0);
+
+    let db_status = value == 1;
+
+    let status = ServiceStatus::new(db_status);
+    HttpResponse::Ok().json(status)
+}
+
+/// Service Health Check
+#[derive(Serialize)]
+struct ServiceStatus {
```

Comment:
Where should we put this struct? @rust-lang-ve/hilow

Reply:
I think the right place for this is here as this is not used in any other place. Lets land this before the other PR, I will solve conflicts and update them